src/main/java/seedu/address/MainApp.java

```diff
@@ -56,8 +56,8 @@ public void init() throws Exception {
 
         UserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(config.getUserPrefsFilePath());
         UserPrefs userPrefs = initPrefs(userPrefsStorage);
-        AddressBookStorage addressBookStorage = new JsonAddressBookStorage(userPrefs.getAddressBookFilePath());
-        storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        CatalogueStorage catalogueStorage = new JsonCatalogueStorage(userPrefs.getCatalogueFilePath());
```

Comment:
Should be Catalog (US spelling) instead of the Catalogue which is UK spelling. CS2103T coding style states to use US spelling.

Reply:
Fixed in latest commit!